src/rpm_ostree/cli_deploy.rs

```diff
@@ -2,19 +2,36 @@
 
 use super::Release;
 use failure::{bail, format_err, Fallible, ResultExt};
+use prometheus::{IntCounter, IntCounterVec};
+
+lazy_static::lazy_static! {
+    static ref DEPLOY_ATTEMPTS: IntCounter = register_int_counter!(opts!(
+        "zincati_rpm_ostree_deploy_attempts_total",
+        "Total number of 'rpm-ostree deploy' attempts."
+    )).unwrap();
+    static ref DEPLOY_RUNS: IntCounterVec = register_int_counter_vec!(opts!(
+        "zincati_rpm_ostree_deploy_runs_total",
+        "Total number of 'rpm-ostree deploy' runs."),
```

Comment:
Maybe `s/runs/results/`? Otherwise, it's not obvious what the difference is between attempt vs run.